README.md

```diff
@@ -486,6 +486,8 @@ Available options (all `false` by default):
 + `async`: Asynchronous execution. If a callback is provided, it will be set to
   `true`, regardless of the passed value.
 + `silent`: Do not echo program output to console.
++ and any option available to NodeJS's
+  [child_process.exec()](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback)
```

Comment:
@ariporad I try to format lines to be under 80 characters. This is how vim's built-in formatting did it. Markdown allows you to extend lines (as long as you don't leave one blank line in-between) without it affecting the end-result. You can check "view file" to be sure